cli/using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,104 @@
+[id="using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: using-roxctl-cli
+
+toc::[]
+
+[id="getting-started_{context}"]
+== Getting started
```

Comment:
Just as a remark: reading through the documentation, specifically the `Using the roxctl CLI` page, we first tell the user to run `roxctl central whoami` and only afterwards give documentation on how they authenticate (including setting up the environment variables holding the endpoint). Shouldn't this come first, and then we'll give the `roxctl central whoami` example?

Alternatively, we can also call out before the `roxctl central whoami` command that setting the `ROX_ENDPOINT` variable is required.

Reply:
Added a new "Prerequisites" section